draft-ietf-quic-ack-frequency.md

```diff
@@ -223,19 +222,15 @@ Request Max Ack Delay:
   milliseconds. Sending a value smaller than the `min_ack_delay` advertised
   by the peer is invalid. Receipt of an invalid value MUST be treated as a
   connection error of type PROTOCOL_VIOLATION.
+  
+Reordering Threshold:
 
-Reserved:
-
-: This field has no meaning in this version of ACK_FREQUENCY.  The value of this
-  field MUST be 0x00. Receipt of any other value MUST be treated as a
-  connection error of type FRAME_ENCODING_ERROR.
-
-Ignore Order:
-
-: A 1-bit field representing a boolean truth value. This field is
-  set to `true` by an endpoint that does not wish to receive an immediate
-  acknowledgement when the peer receives a packet out of order
-  ({{out-of-order}}). 0 represents 'false' and 1 represents 'true'.
+: A variable-length integer representing an unsigned integer that indicates how
+  out of order packets can arrive before eliciting an immediate ACK. If no
```

Comment:
```suggestion
: A variable-length integer representing an unsigned integer that indicates how
  many out of order packets can arrive before eliciting an immediate ACK. If no
```

Reply:
Do we need "A variable-length integer representing an unsigned integer" or can we just rely on the fact that variable-length integers are always unsigned and say:

```suggestion
: A variable-length integer that indicates how many
  out of order packets can arrive before eliciting an immediate ACK. If no
```